<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ForgotPw</class>
 <widget class="QDialog" name="ForgotPw">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>349</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Forgot password</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/img/forgot.png</normaloff>:/img/forgot.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
	background-color:rgb(235, 235, 235);
}

QMessageBox{
	background-color: rgb(255, 255, 255);
	color:rgb(25, 25, 25);
	font: 350 12pt &quot;Bahnschrift SemiLight SemiConde&quot;;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="3">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="2" rowspan="2">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="2">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>Franklin Gothic Heavy</family>
        </font>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; color:#c38200;&quot;&gt;BRY&lt;/span&gt;&lt;span style=&quot; font-size:16pt; color:#0964ba;&quot;&gt;Insurance&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#000000;&quot;&gt;Reset password&lt;/span&gt;&lt;/p&gt;&lt;hr/&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QLineEdit" name="answerLn">
       <property name="minimumSize">
        <size>
         <width>160</width>
         <height>32</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit{
	padding-left:5px;
	color:rgb(238, 238, 238);
	background-color:rgb(26, 26, 26);
	border:0px;
	border-radius:10px;
}

QLineEdit:focus{
	color:rgb(238, 238, 238);
	background-color:rgb(70, 70, 70);
	border:1.5px inset rgb(10, 111, 206);
}</string>
       </property>
       <property name="placeholderText">
        <string>Enter answer:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="keyQuestionLb">
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Bahnschrift SemiLight SemiConde</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
	color:rgb(0, 0, 0);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="4">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item row="7" column="2">
      <widget class="QPushButton" name="confirmPb">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Bahnschrift SemiLight SemiConde</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
border: solid 0px #ffffff;
background-color: rgb(9, 100, 186);
padding-top: 6px;
padding-bottom: 6px;
padding-left: 15px;
padding-right: 15px;
margin-left:15px;
margin-right:15px;
border-radius:15px;
color:#fff;
}

QPushButton:hover{
border: solid 2px rgb(0, 0, 0);
background-color: rgb(195, 130, 0);
}</string>
       </property>
       <property name="text">
        <string>Confirm</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="1">
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Bahnschrift SemiLight SemiConde</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
	color:rgb(0, 0, 0);
}</string>
       </property>
       <property name="text">
        <string>Your key question is:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="0">
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="Line" name="line_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="2">
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
